packages/math/greatest_common_divisor.pony

```diff
@@ -0,0 +1,20 @@
+primitive GreatestCommonDivisor
+  """
+  Get greatest common divisor of x and y
+  """
+
+  fun gcd(x: USize, y: USize): USize ? =>
```

Comment:
According to the code example, this should be an `apply` method so it can get the apply sugar used in the code example.

```suggestion
  fun apply(x: USize, y: USize): USize ? =>
```